examples/make_assets/ptable/ptable_matplotlib.py

```diff
@@ -140,8 +140,7 @@
 for n_splits in (2, 3, 4):
     data_dict = {
         elem.symbol: [
-            random.randint(10 * n_splits, 20 * (n_splits + 1))
-            for _ in range(n_splits)  # noqa: S311
```

Comment:
Looks like a small issue with `ruff-format` here, causing an interlock:
1. It would first unwrap these two lines, but lose the `noqa` mark.
2. After manually adding back the `noqa` mark to the single line, it would get wrapped again.